Client/Frontend/Sync/BraveCore/BraveSyncAPIExtensions.swift

```diff
@@ -158,3 +158,40 @@ extension BraveSyncAPI {
         }
     }
 }
+
+extension BraveSyncAPI {
+    func getQRCodeImageV2(_ size: CGSize) -> UIImage? {
+        let hexCode = hexSeed(fromSyncCode: getSyncCode())
+        if hexCode.isEmpty {
+            return nil
+        }
+
+        // Typically QR Codes use isoLatin1, but it doesn't matter here
+        // as we're not encoding any special characters
+        guard let syncCodeData = BraveSyncQRCodeModel(syncHexCode: hexCode).jsonData,
+              !syncCodeData.isEmpty else {
+            return nil
+        }
+        
+        guard let filter = CIFilter(name: "CIQRCodeGenerator") else {
+            return nil
+        }
+
+        filter.do {
+            $0.setValue(syncCodeData, forKey: "inputMessage")
+            $0.setValue("H", forKey: "inputCorrectionLevel")
```

Comment:
Curious why did we wanna use inputCorrectionLevel, it is not a big image we are using there right ?

Reply:
We want the highest level of error correction in case the QR code is corrupted or can't be read properly.
https://developer.apple.com/documentation/coreimage/ciqrcodedescriptor/errorcorrectionlevel/levelh